vars/pipelineSnapshotTesting.groovy

```diff
@@ -188,18 +201,53 @@ void call(Map config=[:]) {
                                 ./dasel put bool -f tm_config/config/config.toml p2p.allow_duplicate_ip true
                                 cat tm_config/config/config.toml
                             """
+                        sh label: 'set vega config',
+                            script: """#!/bin/bash -e
+                                ./dasel put bool -f vega_config/config/node/config.toml Broker.Socket.Enabled true
+                            """
+
+                        PEERS = sh(
+                            label: 'read persistent peers',
+                            script: './dasel -f data-node-config.toml -r toml DeHistory.BootstrapPeers',
+                            returnStdout: true
+                        ).trim()
+
+                        sh label: 'set data-node config',
+                            script: """#!/bin/bash -e
+                                ./dasel put bool -f vega_config/config/data-node/config.toml AutoInitialiseFromDeHistory true
+                                ./dasel put bool -f vega_config/config/data-node/config.toml SQLStore.UseEmbedded false
+                                ./dasel put string -f vega_config/config/data-node/config.toml SQLStore.ConnectionConfig.Host 127.0.0.1
+                                ./dasel put string -f vega_config/config/data-node/config.toml SQLStore.ConnectionConfig.Port 5432
+                                ./dasel put string -f vega_config/config/data-node/config.toml SQLStore.ConnectionConfig.Username postgres
+                                ./dasel put string -f vega_config/config/data-node/config.toml SQLStore.ConnectionConfig.Password postgres
+                                ./dasel put string -f vega_config/config/data-node/config.toml SQLStore.ConnectionConfig.Database postgres
+                                ./dasel put array -f vega_config/config/data-node/config.toml DeHistory.Store.BootstrapPeers ${PEERS}
+                            """
                     }
 
                     stage('Run') {
                         parallel(
                             failFast: true,
+                            'Postgres': {
+                                nicelyStopAfter(params.TIMEOUT) {
+                                    sh label: 'run postgres',
+                                        script: 'docker run -e POSTGRES_PASSWORD=postgres -p 5432:5432 postgres:14-bullseye'
```

Comment:
Why are we using postgresql instead of the timescaledb?

Is it intended to use it in docker?

Reply:
Yes, we need to use timescaledb (a different docker image), and also configure the database name and user to access it.